Tests/ClientRuntimeTests/NetworkingTests/Http/SdkRequestBuilderTests.swift

```diff
@@ -18,7 +18,7 @@ class SdkRequestBuilderTests: XCTestCase {
         let crtRequest = try HTTPRequest()
         crtRequest.path = pathToMatch
 
-        let updatedRequest = HTTPRequestBuilder().update(from: crtRequest, originalRequest: originalRequest).build()
+        let updatedRequest = HTTPRequestBuilder().update(from: crtRequest, originalRequest: originalRequest, signedAt: Date()).build()
```

Comment:
A `signedAt` value had to be passed to allow compile.  It is not used in this test.